README.md

```diff
@@ -23,19 +23,36 @@ The W3C itself must function with these strategic principles:
 
 * Put the needs of users first: above authors, publishers, implementers, paying W3C Members, or theoretical purity
 * Ensure that the Web can work for everyone, striving for diversity & inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more. Continue to champion accessibility and internationalization as core principles for the Web.
-* Make the web trustworthy, by improving security and privacy for users. We must revisit the current Web platform and explore how to responsibly improve personal privacy for users.
+* Make the web trustworthy, by improving security and privacy for users. We must revisit the current Web platform and explore how to responsibly improve personal privacy for users. 
```

Comment:
we could avoid the split infinite and make it more active by saying "how we can responsibly improve personal privacy"